src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -85,7 +95,11 @@ protected String getPropertyName() {
 	@Override
 	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) throws UnsupportedOperationException {
 		assert delta != null;
-		final Location l = getExpr().getSingle(e);
+		Object o = getExpr().getSingle(e);

```

Comment:
I don't know why this expression only supported a single expression to be changed but it should definitely support multiple locations/chunks to be changed.